src/test/components/ButtonReset.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
+
+import { ButtonReset } from '../../forms/ButtonReset';
+
+import { CLIENT_DEFAULTS } from '../../utils/CLIENT_DEFAULTS';
```

Comment:
It doesn't look like `CLIENT_DEFAULTS` or `renderer` is used in this file. Can these be removed?

Reply:
Wonder why the tests aren't catching that. It's supposed to catch warnings like 'unused vars' last I checked. Wonder if/when that changed. Regardless, it doesn't break anything.